src/audiomixerboard.cpp

```diff
@@ -1070,9 +1070,15 @@ void CAudioMixerBoard::ChangeFaderOrder ( const EChSortType eChSortType )
     // create a pair list of lower strings and fader ID for each channel
     QList<QPair<QString, int>> PairList;
     int                        iNumVisibleFaders = 0;
+    int                        iMyFader          = -1;
 
     for ( int i = 0; i < MAX_NUM_CHANNELS; i++ )
```

Comment:
@ngocdh 
There might be issues with the code here (see #2738) can you please look at this again?

Reply:
(And note, again, I'd prefer if all this "state" information was held by the client - the UI should be responsible for no more than displaying that information as held.)